courses/constants.py

```diff
@@ -25,3 +25,12 @@
 
 COURSE_BG_IMG_MOBILE_W_H = (1024, 350)
 COURSE_BG_IMG_MOBILE_WAGTAIL_FILL = "fill-{}x{}".format(*COURSE_BG_IMG_MOBILE_W_H)
+
+
+class FinalGradeStatus:
+    """
+    Possible statuses for the Final Grades
+    """
+    PENDING = 'pending'
+    COMPLETE = 'complete'
+    ALL_STATUSES = [PENDING, COMPLETE]
```

Comment:
Can you please confirm if there can be any other status as well?

Reply:
Currently, In micro-master I saw these two. we can add as we proceed further.